Makefile

```diff
@@ -5,6 +5,9 @@ DOCKERBUILD := docker build --build-arg http_proxy=$$http_proxy --build-arg http
 
 PIP?=pip
 
+VENV_NAME:=.venv$(VENV_PY_VERSION)
```

Comment:
Maybe I didn't understand you correctly.

What should I fix?
If no python version is specified (simply make virtualenv)  then $VENV_PY_VERSION is not
defined when we define VENV_NAME. 
So VENV_NAME is set to .venv because we use ":=" assignment. In this case variables are 
expanded immediately. 
If it was  "=" assignment then variables would expand recursively. So it would be
.venvpython because later we define VENV_PY_VERSION (set it to python if not defined).

I did so because I thought what it is easier than several ifs.
Maybe I should add comment about usage with no python version specified